### DISCUSSION TITLE
Koşu yarışmasının ilk üçü sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#20] [Zeka Sorusu]
10 kişinin katıldığı ve Merve’nin 4. olduğu bir koşuda, yarışı ilk üç sırada bitirenler kaç farklı şekilde belirlenir?

**Comment by HabilCcabbar** [comment]
[YarışmaSıralası.xlsx](https://github.com/sonsuzus/sonsuzus/files/12095686/YarismaSiralasi.xlsx)

**Comment by begumasyaburus** [comment]
1. Sıraya merve dışında gelebilecek 9 kişi var 

2. Sıraya merve dışında gelebilecek 8 kişi var 

3. Sıraya merve dışında gelebilecek 7 kişi var 
 
4. Merve

5. Sıraya merve dışında gelebilecek 6 kişi var 

Ve bu şekilde devam edersek 10. Sıraya gelebilecek 1 kişi kalır 
Sıra sayılarını birbirleri ile çarparsak yani 9.8.7.6...1  = 362880 buluruz.
